task-submission-service/Dockerfile

```
@@ -0,0 +1,18 @@
+FROM eclipse-temurin:21-jdk-alpine
+
+ARG JAR_FILE=target/*.jar
+
+# Create upload directory with correct permissions
+RUN mkdir -p /uploads/submission && \
+    chown -R 1000:1000 /uploads && \
+    chmod -R 775 /uploads
+
+COPY ${JAR_FILE} submissionservice.jar
```

Hadolint (2.12.0)

[warning] `COPY` to a relative destination without `WORKDIR` set.

(DL3045)